docs/UserGuide.md

```diff
@@ -427,13 +446,13 @@ Example:
 ## FAQ
 
 **Q**: How do I save changes?
-**A**: Athena's data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
-
-**Q**: Where are the save files that Athena uses?<br>
-**A**: The default save directory is in `{Athena home directory}/data`.
+**A**: Athena's data is saved in the hard disk automatically after any command that changes the data. There is no need to manually save the data.
```

Comment:
"There is no need to save manually." is more succinct and less awkward than "There is no need to manually save the data.".  The inherent meaning is the same.